### PR TITLE
fix > character jump/climbing steep slopes

### PR DESCRIPTION
**Pull request by Mdotta**
I fixed the problem while implementing sliding down the slopes.

### CODE REVIEW

**Comment by ciro-unity** [comment]
There's also another PR open that claims to fix this issue: #12

File: UOP1_Project/Assets/Scenes/CharController.unity [hunk 1]
```diff
@@ -1059,6 +1059,36 @@ Transform:
...
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 880196732}
+  m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
+  m_LocalPosition: {x: 0, y: 1.891, z: 0}
+  m_LocalScale: {x: 0.28572452, y: 0.1339436, z: 0.45457}
+  m_Children: []
+  m_Father: {fileID: 1961901426}
+  m_RootOrder: 5
+  m_LocalEulerAnglesHint: {x: 0, y: 0, z: 0}
```
Comment (ciro-unity) [inline UOP1_Project/Assets/Scenes/CharController.unity:1091]: Just FYI, we can't accept PRs that contain thousands of changed lines unrelated to the issue.

Comment (Mdotta) [reply UOP1_Project/Assets/Scenes/CharController.unity:1091]: I made a change in the scene adding one object to test slopes, I can commit just the script but how should I update this PR after it?

**Comment by Mdotta** [comment]
> There's also another PR open that claims to fix this issue: #12

I saw this PR when I was about to send mine. Both PRs fixes the issue but I fixed it with a "sliding down the slope" mechanic

**Pull request closed without merging**